### ISSUE TITLE
elm make --warn --yes flags absent in 0.19

### ISSUE THREAD

**Issue by lmmarsano** [elm/compiler#1752]
For elm 0.18, emacs was running `elm-make --yes --warn --output=temp-file-output --report=json temp-file-source` to read & fill out type annotations.
When I try to adapt these commands to elm 0.19, I notice flags `--yes --warn` are no longer accepted.
```ShellSession
$ elm make --yes --warn --output=/dev/null --report=json $file
I do not recognize this flag:

    --yes

$ elm make --warn --output=/dev/null --report=json $file
I do not recognize this flag:

    --warn
```
Is there a replacement for this functionality or a better way to get type annotations?

**Comment by evancz** [comment]
Those flags did not get into 0.19. We will bring warnings back at some point in some form, but with the compiler rewrites, it was quite difficult and I did not want to block the release any longer.

For editor plugins that relied on flags for unintended uses, I do not have good recommendations. We would like to support things explicitly with actual APIs at some point, but we have not had support for that so far.

**Comment by flicaflow** [comment]
What about the --yes flag , it missing just broke our CI

**Comment by mordrax** [comment]
@bruchmann we're using vsts as our CI and putting `yes | ` in front seems to cause that task to run indefinitely.
